### PR TITLE
Listing Controller and View

### PR DESCRIPTION
**Pull request by LexLPS**
Added Controller and View with routes inside routes.rb (recources) commented out the normal ones.

### CODE REVIEW

File: app/controllers/listings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  def show
+  end
+
+  def create
+    @listing = Listing.new(listing_params)
+
+    if @listing.save
+      redirect_to listing_path(@listing)
+    else
+      render 'new'
```
Comment (levelev) [inline app/controllers/listings_controller.rb]: missing end

File: app/controllers/listings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  end
+
+  def edit
+  end
+
+  def update
+    if @listing.update(listing_params)
+      redirect_to @listing
+    else
+      render 'edit'
```
Comment (levelev) [inline app/controllers/listings_controller.rb]: missing end

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+
+  # get 'listings/create'
+
+  # get 'listings/new'
+
+  # get 'listings/edit'
+
+  # get 'listings/update'
+
+  resources :listings, only: [:index, :show, :new, :edit, :update, :create]
```
Comment (levelev) [inline config/routes.rb]: `except: [:destroy]`

File: app/controllers/listings_controller.rb (hunk 1 above)
Comment (LexLPS) [reply app/controllers/listings_controller.rb]: √

File: app/controllers/listings_controller.rb (hunk 2 above)
Comment (LexLPS) [reply app/controllers/listings_controller.rb]: √

File: config/routes.rb (hunk 3 above)
Comment (LexLPS) [reply config/routes.rb]: √

**Pull request merged**